test/samples/meson.build

```diff
@@ -68,7 +68,43 @@ foreach name, cmd: samples
     env = name == 'locale.mkv' ? 'LANG=C' : []
     testname = 'libmpv-test-' + name
     exe = executable(testname, '../libmpv_test_track_selection.c',
-                     dependencies: libmpv_dep)
+                     c_args: libmpv_flags, dependencies: libmpv_dep)
     test(testname, exe, args: [name, target.full_path()],
          depends: target, env: env, suite: 'libmpv')
 endforeach
+
+
+sub_sdh = join_paths(source_root, 'test', 'samples', 'sub_sdh.srt')
+sub_mixed = join_paths(source_root, 'test', 'samples', 'sub_mixed.srt')
+
+sdh_samples = {
+    'sdh_default.mkv':
+        [ffmpeg, '-v', 'error', '-y', '-i', audio, '-f', 'srt',
+        '-i', sub_sdh, '-map', '0:0', '-map', '1:0', '-disposition:s:0',
+        'default', '@OUTPUT@'],
+    'sdh_harder.mkv':
```

Comment:
do we need separate file this? It looks the same as above, no? I think you could add additional field with test file name and test name to reuse filenames.

Reply:
We don't really but it's kind of nice to have separate files since it gives you a separate test on every thread.

Reply:
file generation step is separate from test step. You can use the same file input for multiple tests that will work in parallel.